lymph/midline.py

```diff
@@ -150,10 +190,89 @@ def trans_probs(self, new_params: np.ndarray):
         """
         self.noext.trans_probs = new_params
         self.ext.trans_probs = new_params
-
+    
         # avoid unnecessary double computation of ipsilateral transition matrix
         self.noext.ipsi._transition_matrix = self.ext.ipsi.transition_matrix
 
+    def _gen_allowed_midextransitions(self):
```

Comment:
Seems a bit unnecessary, no? I mean the entire transition matrix for the midline extension state is a 2x2 matrix with one entry that is always zero 😅

Reply:
Absolutely, I'll "hand-write" it:)